controllers/secrets-webhook/secretswebhook.go

```diff
@@ -61,11 +64,11 @@ func (p *podSecretsInjector) Handle(ctx context.Context, req admission.Request)
 	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledPod)
 }
 
-// e.g. k8ssandra.io/inject-secret: '[{ "secretName": "test-secret", "path": "/etc/test/test-secret" }]'
+// e.g. k8ssandra.io/inject-secret: '[{ "name": "test-secret", "path": "/etc/test/test-secret" }]'
 const secretInjectionAnnotation = "k8ssandra.io/inject-secret"
 
 type SecretInjection struct {
-	SecretName string `json:"secretName"`
+	SecretName string `json:"name"`
```

Comment:
The ticket also suggests that the old name still be supported for backward compatibility. It looks like there's no way to have multiple name tags, so we could have both fields and pick the first non nil.

Although that raises the question of whether changing the name in the first place is worth it...

Reply:
I'm to blame for trying to keep some backwards compatibility here. It's probably not worth it and we can move forward with just `name`.